# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mjpegtools/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mjpegtools-1.8.0/y4mdenoise/Region2D.hh_orig	2006-03-04 12:59:59.574110040 +0100
+++ mjpegtools-1.8.0/y4mdenoise/Region2D.hh	2006-03-04 13:01:09.642458032 +0100
@@ -18,6 +18,7 @@
 #include <iostream>
 
 
+#include <cassert>
 
 // The 2-dimensional region class.  Parameterized by the numeric type
 // to use for point indices, and the numeric type to use to count the
